README.md

```diff
@@ -1,16 +1,22 @@
 # omsoo-portfolio
 
-This is my portfolio page that shows my skills, experience and history of works done. 
-
+The project is still in the works, but when completed, it will tell my story, skills, experience, and work history. Constructive feedback is encouraged.
 
 ## Built With
 
 - HTML
 - CSS
```

Comment:
- [ ] You did a great job adding the README file but as the previous reviewer said it should have a section that tells a user how to set up and run your project locally on their machine. Please use [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) for guidance. Keep rocking 💪